pkg/providers/cloudstack/decoder/decoder.go

```diff
@@ -13,6 +13,8 @@ const (
 	EksacloudStackCloudConfigB64SecretKey = "EKSA_CLOUDSTACK_B64ENCODED_SECRET"
 	CloudStackCloudConfigB64SecretKey     = "CLOUDSTACK_B64ENCODED_SECRET"
 	EksaCloudStackHostPathToMount         = "EKSA_CLOUDSTACK_HOST_PATHS_TO_MOUNT"
+	globalSectionName                     = "Global"
+	defaultSectionName                    = "DEFAULT"
```

Comment:
why do we need this? If every ini file must have a DEFAULT section, could we use an existing constant instead of defining our own?